mlir/test/Dialect/SCF/canonicalize.mlir

```diff
@@ -1,766 +1,6 @@
 // RUN: mlir-opt %s -pass-pipeline='builtin.module(func.func(canonicalize{test-convergence}))' -split-input-file | FileCheck %s
 
-func.func @single_iteration_some(%A: memref<?x?x?xi32>) {
```

Comment:
Can you provide more context why all these tests have to be removed?
Is it related to the index issue?